docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -35,26 +35,26 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 * EE2 looks up the job versions and parameters, and then submits the job to be retried, incrementing the `retry_count`
   of the job being retried, and the newly launched job gains a pointer to the `_PARENT_RETRY_JOB_ID`
```

Comment:
does `_PARENT_RETRY_JOB_ID` refer to the name of the field in mongodb? The terminology should be consistent, so this should refer to assigning the `retry_parent` of the newly-launched job, rather than pointers and db field names.

Reply:
updated